app/models/ability.rb

```diff
@@ -0,0 +1,59 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
```

Comment:
da admin und representative immer wimiws sind, kann man das irgendwie zusammen ausführenlassen?
quasi die funktion so:

initialize_admin user if user.is_admin?
initialize_representative user if user.is_representative?
initialize_wimi user and return if user.is_wimi?

initialize_superadmin user and return if user.is_superadmin?
initialize_hiwi user and return if user.is_hiwi?
initialize_user user and return if user.is_user?